nipoppy_cli/tests/data/dicom_dir_map2.csv

```diff
@@ -1,4 +1,4 @@
-participant_id,session,participant_dicom_dir
-01,ses-BL,all_BL_files_are_together
-01,ses-M12,all_M12_files_are_together
-02,ses-BL,all_BL_files_are_together
+participant_id,session_id,participant_dicom_dir
+01,BL,all_BL_files_are_together
```

Comment:
why is this a valid map?

Reply:
Because the test doesn't care about the actual paths, just about general validity of the dataframe :)